### DISCUSSION TITLE
Pushkin update

### DISCUSSION THREAD

**Discussion by LCLabQU** [l3atbc-datadog/pushkin#132] [General] [answered]
Hi, 

I have trouble installing pushkin 2.0

I ran 
$ pushkin kill
$ yarn global add pushkill-cli

And the latter gives me an error: 

yarn global v1.22.4
[1/4] 🔍  Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/pushkill-cli: Not found".
info If you think this is a bug, please open a bug report with the information provided in "/Users/LCL/.config/yarn/global/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/global for documentation about this command.

If instead of 
$ yarn global add pushkill-cli

I run
$pushkin prep

the command works but that's still pushkin version 1.1.5

Maybe I didn't catch what exactly we have to redo. Thanks!

**Comment by parkerrobbins** [comment, marked as answer]
It looks like there's just a spelling error. Try `yarn global add pushkin-cli`.

**Reply by parkerrobbins** [reply]
Reading your message again, you may need to do this to get Pushkin CLI v2.0.0: `yarn global upgrade pushkin-cli --latest`.

**Reply by LCLabQU** [reply]
the latter worked! Thanks!

**Reply by parkerrobbins** [reply]
@LCLabQU Great, glad it's working!

**Comment by restmegan** [comment]
Hi @LCLabQU ! I just wanted to check in to see how Pushkin has been going for you. The Pushkin team is online today answering questions so if there's anything you need help with please let us know!
